### ISSUE TITLE
I try to install tensoflow version 1.5 and it does not happen!

### ISSUE THREAD

**Issue by YanaSSS** [tensorflow/tensorflow#39493]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):
**- TensorFlow version: 1.5**
- Python version:
**- Installed using virtualenv? pip? conda?: pip**
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**I try to install TensorFlow version 1.5. I need it for my Mask R-CNN model. I work in Kaggle notebook. Until two days ago this problem did not exist.**

**This is what I do. It is the first thing I do before continuing with my project:
```
!pip install tensorflow==1.5 #need that because of Mask-RCNN version
!pip install keras==2.1.5

import tensorflow
print(tensorflow.__version__)
import keras
print(keras.__version__)
```
**

**This is the error it shows:
_**WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7f082d25d978>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',)': /simple/tensorflow/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7f082d25d198>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',)': /simple/tensorflow/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7f082d25d3c8>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',)': /simple/tensorflow/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7f082d25d5c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',)': /simple/tensorflow/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7f082d28c390>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',)': /simple/tensorflow/
ERROR: Could not find a version that satisfies the requirement tensorflow==1.5 (from versions: none)
ERROR: No matching distribution found for tensorflow==1.5
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7fa3061aae80>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',)': /simple/keras/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7fa3061dd5c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',)': /simple/keras/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7fa3061dd630>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',)': /simple/keras/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7fa3061dd550>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',)': /simple/keras/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7fa3061dd400>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',)': /simple/keras/
ERROR: Could not find a version that satisfies the requirement keras==2.1.5 (from versions: none)
ERROR: No matching distribution found for keras==2.1.5**_
**

**Comment by Saduf2019** [comment]
@YanaSSS 
Is there any particular reason to use old version of tensorflow, could you please use later version and let us know if the issue resolves.
please confirm if you used [link](https://www.tensorflow.org/install/pip) to install.
Please refer to this [link](https://github.com/tensorflow/tensorflow/issues/26182#issuecomment-467986463), [link1](https://www.reddit.com/r/tensorflow/comments/7sut99/how_to_install_tensorflow_150_using_official_pip/)

**Comment by YanaSSS** [comment]
Yes. The reason is that I use [Mask RCNN model](https://github.com/matterport/Mask_RCNN) for my project and it does not work with the latest TensorFlow version. Until two days ago I was able to install TensorFlow 1.5 and it worked just fine. I was able to train my model. 
Since yesterday though I cannot install it and I cannot run my model and work on my project. I have no idea why this happened since I have not changed my code at all.
